geotrek/feedback/forms.py

```diff
@@ -15,8 +41,46 @@ class Meta:
             "problem_magnitude",
             "related_trek",
             "status",
-            "locked",
-            "uid",
-            "origin"
+            "assigned_user"
         ]
         model = Report
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["geom"].required = True
+        if settings.SURICATE_MANAGEMENT_ENABLED:
+            if self.instance.pk:
+                # Store current status
+                self.old_status_id = self.instance.status.suricate_id
```

Comment:
```suggestion
                self.old_suricate_id = self.instance.status.suricate_id
```

status_id != suricate_id